cob_object_categorization/CMakeLists.txt

```diff
@@ -1,153 +1,109 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 2.8.3)
 project(cob_object_categorization)
 
-set(catkin_RUN_PACKAGES 
-	pcl_ros # Should automatically pull in all PCL dependecies
-	roscpp
-	cv_bridge
-	sensor_msgs
-	message_filters
-	image_transport
-	nodelet
-	cob_perception_msgs
-	cob_object_detection_msgs
-	tf
-	actionlib
-	actionlib_msgs
-)
-
-set(catkin_BUILD_PACKAGES 
-	${catkin_RUN_PACKAGES}
-	cmake_modules
-)
-
-
-## Find catkin macros and libraries
-## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
-## is used, also find other catkin packages
-## Creates a bunch of environment variables that may be used later in the script
-## e.g. catkin_INCLUDE_DIRS
 
 find_package(catkin REQUIRED COMPONENTS
-	${catkin_BUILD_PACKAGES}
+  actionlib
+  actionlib_msgs
+  cmake_modules
+  cob_object_detection_msgs
+  cob_perception_msgs
+  cv_bridge
+  image_transport
+  message_filters
+  nodelet
+  pcl_ros
+  roscpp
+  sensor_msgs
+  tf
 )
 
+find_package(Eigen REQUIRED)
 find_package(OpenCV REQUIRED)
 #find_package(PCL REQUIRED)
-find_package(Eigen REQUIRED)
 find_package(VTK REQUIRED)
 
-###################################
-## catkin specific configuration ##
-###################################
-## The catkin_package macro generates cmake config files for your package
-## Declare things to be passed to dependent projects
-## INCLUDE_DIRS: uncomment this if you package contains header files
-## LIBRARIES: libraries you create in this project that dependent projects also need
-## CATKIN_DEPENDS: catkin_packages dependent projects also need
-## DEPENDS: system dependencies of this project that dependent projects also need
-catkin_package(
-INCLUDE_DIRS
-	common/include
-LIBRARIES
-CATKIN_DEPENDS
-	${catkin_RUN_PACKAGES}
-DEPENDS
```

Comment:
Why is the catkin_package block not needed here? What about the depends libraries?